README.md

```diff
@@ -108,4 +114,21 @@ go get -u github.com/sourcegraph/src-cli/cmd/src
 3.  Travis will automatically perform the release. Once it has finished, **confirm that the curl commands fetch the latest version above**.
 4.  Update the `MinimumVersion` constant in the [src-cli package](https://github.com/sourcegraph/sourcegraph/tree/master/internal/src-cli/consts.go).
 
-**Note**: The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+### Patch releases
+
+If a backwards-compatible change is made _after_ a backwards-incompatible one, the backwards-compatible one should be re-released to older instances that support it.
+
+A Sourcegraph instance returns the highest patch version with the same major and minor version as `MinimumVersion` as defined in the instance. Patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of `src-cli` without having to update the instance.
+
+To release a bug fix or a new feature that is backwards compatible with one of the previous two minor version of Sourcegraph, cherry-pick the changes into a patch branch and re-releases with a new patch version. 
+
+For example, suppose we have the the recommended versions.
+
+| Sourcegraph version | Recommended src-cli version |
+| ------------------- | --------------------------- | 
+| `3.100`             | `3.90.5`                    |
+| `3.99`              | `3.85.7`                    |
+
+If a new feature is added to a new `3.91.6` release of src-cli and this change requires only features available in Sourcegraph `3.99`, then this feature should also be present in a new `3.85.8` release of src-cli. Similar logic should follow for Sourcegraph `3.98` and its recommended version of src-cli. Because a Sourcegraph instance will automatically select the highest patch version, all non-breaking changes should increment only the patch version. 
```

Comment:
I thought the logic was only for one rev +/-, is that not the case?

Reply:
Oh yes we've updated it. I'll make the change.